features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -968,6 +968,10 @@
         </Claim>
     </UserClaimUpdate>
 
+     <AccountSuspension>
+        <UseIdentityClaims>{{account_suspension.last_login_as_identity_claim}}</UseIdentityClaims>
```

Comment:
let's change this as identity_mgt.account_suspension.last_login_as_identity_claim similar to other